live-build/misc/ansible-roles/appliance-build.zfsonlinux-development/tasks/main.yml

```diff
@@ -65,6 +65,39 @@
     accept_hostkey: yes
     update: no
 
+- shell: ls /lib/modules
+  register: kernel_versions
+
+- shell: |
+    CONFIG_OPTS=" \
```

Comment:
@pzakha could we move this into the zfs repository some how? so both git-utils and this code wouldn't have to do this? do we also have to do this for the zfs package build?

Reply:
These specific options are unique to delphix's use case, other users will probably have their own requirements for any such script, so it's not likely that such a script would land upstream. We could have it live in delphix/zfs and maintain the fork, which would probably not be prohibitively difficult. That would eliminate the need to store multiple copies of these options.  I'm not sure exactly how we do it in the zfs package build.

Reply:
We could definitely move this into the Delphix ZFS repository if we wanted. I'm assuming you got most of those options from git-zol-make. Originally, those were actually taken from the `debian/rules` in order to make the manual build as close as possible to the package build.

So I think if we add this to the internal-dev variant, the main issue is that it will add 20 minutes to all git-ab-pre-push runs. This may be fine, but we should probably have a discussion whether we want to go that route or not. Same goes for the app build.

Reply:
Right. I'm thinking of something that'd we'd land to delphix/zfs; e.g. changing the default values, or something similar.

I'm sure we could create some sort of "delphix-configure" script, with these values baked in, but I'm not sure if that's the best solution we can come up with.

Spreading the same values across different repositories (here, git-utils, etc) would become an issue when/if we need to change these, so if we could consolidate that'd be good.